xapian-applications/omega/templates/query

```diff
@@ -149,7 +160,14 @@ ${Preserve any B filters used - this needs modifying if you modify this template
   to have B on radio buttons, SELECT, etc}
 $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </form>
-<hr><div align=right><i><small><a href="https://xapian.org/">$html{$version}</a></small></i></div>
+<hr>
+<footer align=right>
+<emp>
```

Comment:
You still don't need it; emphasis only makes sense within running text (prose), which this is not. It's not being emphasised compared to anything around it, so just drop it.